client/packages.go

```diff
@@ -65,6 +65,7 @@ type Snap struct {
 	CommonIDs        []string      `json:"common-ids,omitempty"`
 	MountedFrom      string        `json:"mounted-from,omitempty"`
 	CohortKey        string        `json:"cohort-key,omitempty"`
+	InhibitedRefresh bool          `json:"inhibited-refresh,omitempty"`
```

Comment:
after further thinking, we should probably return a time instead of a boolean that matches when the refresh will happen forcefully:

RefreshInhibitedProceedTime *time.Time `json:"refresh-inhibited-proceed-time,omitempty"`

or /Proceed/Force/?

this would be set to SnapState.RefreshInhibitedTime+maxInhibition, so there should be some helper in snapstate to compute it